server/status_test.go

```diff
@@ -219,3 +220,59 @@ func TestStatusGossipJson(t *testing.T) {
 		}
 	}
 }
+
+// TestNodeStatusResponse verifies that node status returns the expected
+// results.
+// TODO(Bram): Add more nodes.
+func TestNodeStatusResponse(t *testing.T) {
+	ts := &TestServer{}
+	ts.Ctx = NewTestContext()
+	ts.Ctx.ScanInterval = time.Duration(5 * time.Millisecond)
+	ts.StoresPerNode = 3
+	if err := ts.Start(); err != nil {
+		t.Fatal(err)
+	}
+	defer ts.Stop()
+
+	// Make sure the node is spun up.
+	ts.node.waitForScanCompletion()
+	ts.node.waitForScanCompletion()
+
+	httpClient, err := testContext.GetHTTPClient()
+	if err != nil {
+		t.Fatal(err)
+	}
+	req, err := http.NewRequest("GET", testContext.RequestScheme()+"://"+ts.ServingAddr()+statusNodesKeyPrefix, nil)
```

Comment:
don't we have a standard http client somewhere? If we do better to use that one, since we might be enforcing client certs, ... in tests as well.